lib/core/sync_job_runner.rb

```diff
@@ -8,6 +8,7 @@
 
 require 'connectors/connector_status'
```

Comment:
This file got out of hand, we'll need to refactor it in close future (outside of scope of this PR)

Reply:
:high_five (it doesn't work 😅 )

https://elastic.slack.com/archives/C01795T48LQ/p1668532145494359?thread_ts=1668532061.722469&cid=C01795T48LQ